docs/socket-activation.rst

```diff
@@ -2,10 +2,11 @@ Socket Activation
 -----------------
 
 While waitress does not support the various implementations of socket activation,
-for example using systemd or launchd, it is prepared to receive pre-bound applications
-from an init system.
+for example using systemd or launchd, it is prepared to receive pre-bound sockets
+from init systems, process and socket managers or other launchers that can provide
+pre-bound sockets.
```

Comment:
There's a comma missing somewhere in this sentence. Perhaps this? Oxford comma FTW!

```
from init systems, process and socket managers, or other launchers that can provide
pre-bound sockets.
```